content/docs/2.4/concepts/scaling-deployments.md

```diff
@@ -121,6 +124,28 @@ This setting is passed to the HPA definition that KEDA will create for a given r
 
 ---
 
+```yaml
+  fallback:                                          # Optional. Section to specify fallback options
+    failureThreshold: 3                              # Mandatory if fallback section is included
+    replicas: 6                                      # Mandatory if fallback section is included
+```
+
+The `fallback` section is optional. It defines a number of replicas to fallback to if a scaler is in an error state.
+
+KEDA will keep track of the number of consecutive times each scaler has failed to get metrics from its source. Once that value passes the `failureThreshold`, instead of not propagating a metric to the HPA (the default error behaviour), the scaler will, instead, return a normalised metric using the formula:
+```
+target metric value * fallback replicas
+```
+Due to the HPA metric being of type `AverageValue` (see below), this will have the effect of the HPA scaling the deployment to the expected number of replicas.
```

Comment:
I think this is what you mean, right? Maybe better to make it a bit more explicit because you could also read it as the general replica number.
```suggestion
Due to the HPA metric being of type `AverageValue` (see below), this will have the effect of the HPA scaling the deployment to the defined number of fallback replicas.
```